MatrixKit/Models/Room/MXKRoomDataSource.m

```diff
@@ -2283,27 +2283,13 @@ - (void)handleUnsentMessages
     NSArray<MXEvent*>* outgoingMessages = _room.outgoingMessages;
     
     [self.mxSession decryptEvents:outgoingMessages inTimeline:nil onComplete:^(NSArray<MXEvent *> *failedEvents) {
-        BOOL shouldProcessQueuedEvents = NO;
         
-        for (NSInteger index = 0; index < outgoingMessages.count; index++)
+        for (MXEvent *outgoingMessage in outgoingMessages)
```

Comment:
Can we log the number of outgoing events we push?

Reply:
Added